std/file.d

```diff
@@ -4616,7 +4616,7 @@ enum SpanMode
     root.buildPath("animals", "dog").mkdir;
     root.buildPath("plants").mkdir;
 
-    alias removeRoot = (e) => e.relativePath(root);
+    alias removeRoot = (return scope e) @trusted => e.relativePath(root);
```

Comment:
I need help trying to figure out the errors here. Starting with this one failing as 

```
std/file.d(4619): Error: returning `relativePath(e.name(), delegate string() pure nothrow @nogc @safe => root)` escapes a reference to parameter `e`
std/file.d(4619):        perhaps annotate the parameter with `return`
std/algorithm/iteration.d(524):        instantiated from here: `MapResult!(__lambda2, DirIterator)`
std/file.d(4621):        instantiated from here: `map!(DirIterator)`
```